### PR TITLE
Additional fixes: src and doc attachment on release.

### PR DESCRIPTION
**Pull request by andresrc**
@reviewbybees

### CODE REVIEW

**Comment by amuniz** [comment]
:bee:

File: pom.xml [hunk 1]
```diff
@@ -772,6 +772,32 @@
...
       </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-source-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>attach-sources</id>
+                <goals>
+                  <goal>jar-no-fork</goal>
```
Comment (jglick) [inline pom.xml]: Yet you are not using `javadoc-no-fork`—intentional?

The [superpom](https://maven.apache.org/ref/3.3.3/maven-model-builder/super-pom.html) seems to use `jar` for both. I do not understand the fine points here, just asking if there is a reason you are diverging from the superpom.

**Comment by jglick** [comment]
:bee: AFAIUI

**Comment by recena** [comment]
:bee:

**Comment by jglick** [comment]
:bee: if you can rerun the same local tests as before.

**Comment by andresrc** [comment]
Tests repeated and ok. Merging...

**Pull request merged**